src/common/class.wpes-wp-post-field-builder.php

```diff
@@ -534,6 +534,127 @@ public function get_mappings( $args = array() ) {
 						),
 					),
 				),
+				'attachment' => array(
```

Comment:
@gibrown I wasn't sure whether it is a great idea to fetch and store this kind of information for all images that can end up in the post. I was especially wondering if this makes sense for images hosted elsewhere.

If there aren't any performance concerns, I totally agree that we should include this in the `image` mapping, which would then need to become a nested object.